src/ser_comm.cpp

```diff
@@ -175,26 +202,14 @@ void SerComm::refresh_ports()
 
 void SerComm::_get_property_list(List<PropertyInfo> *r_list) const
 {
-	r_list->push_back(PropertyInfo(Variant::BOOL, "toggle_to_refresh"));
 	r_list->push_back(PropertyInfo(Variant::INT, "port", PROPERTY_HINT_ENUM, VariantHelper::_ports_to_hint_string(_ports)));
-	r_list->push_back(PropertyInfo(Variant::INT, "baud_rate", PROPERTY_HINT_ENUM, VariantHelper::_baud_rate_to_hint_string()));
 }
 
 bool SerComm::_get(const StringName &p_name, Variant &r_value) const
```

Comment:
Now that we have separate setter and getter for port, baudrate and toggle_to_refresh, do we need the _get?
Don't remember why I did it this way.